hpke/vectors_test.go

```diff
@@ -389,6 +389,10 @@ func TestHybridKemRoundTrip(t *testing.T) {
 			panic("unsupported mode")
 		}
 
+		if rnd2.Len() != 0 {
+			t.Fatal()
+		}
+
```

Comment:
```suggestion
```

It doesn't seem like this is needed since we always create it as 64 bytes up above?

Reply:
This is defensive programming, to make sure we don't have a mismatch between the amount of randomness in the buffer and the actual amount consumed. It's not necessary now, but it might be once we check the upstream test vectors and copy-paste this code.